src/components/calculator.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
+import './calculator.css';
+
+function Calculator() {
+  const [value, setValue] = useState(0);
+
+  return (
+    <div className="container">
+      <div className="output">{value}</div>
+      <div className="buttons">
+        <button type="button">AC</button>
+        <button type="button">+/-</button>
+        <button type="button">%</button>
+        <button type="button" className="operator">÷</button>
+        <button type="button">7</button>
+        <button type="button">8</button>
+        <button type="button">9</button>
+        <button type="button" className="operator">x</button>
+        <button type="button">4</button>
+        <button type="button">5</button>
+        <button type="button">6</button>
+        <button type="button" className="operator">-</button>
+        <button type="button">1</button>
+        <button type="button">2</button>
```

Comment:
- [ ] As suggested by the previous reviewers, kindly consider to have a child component for this Calculator component. what you can do is create a button component and then import it here in the Calculator component. 
you will be sending some props to the button component from Calculator component. its not a difficult tast, I know that you can do it.
  requirement:
  - Create at least 1 child component